README.rst

```diff
@@ -25,6 +25,18 @@ tag.
 
 **OpenFAST is under active development**.
 
+
+
+Part of the WETO Stack
+----------------------
+
+WISDEM is primarily developed with the support of the U.S. Department of Energy and is part of the `WETO Software Stack <https://nrel.github.io/WETOStack>`_. For more information and other integrated modeling software, see:
```

Comment:
should this say `OpenFAST is primarily developed...`

Reply:
Yikes!  I thought I had changed that. . .